gutenberg/_domain_model/text.py

```diff
@@ -93,11 +92,11 @@
 )))
 
 
-LEGALESE_START_MARKERS = frozenset((u(_) for _ in (
+LEGALESE_START_MARKERS = frozenset((_ for _ in (
```

Comment:
Nit: we don't need the `_ for _ in` anymore. We can just initialize the set directly. I love that this commit is super clean (just remove the references to `u`), so this would be a good fix in a second commit.